template-for-repository/proofs/Makefile.common

```diff
@@ -222,6 +222,8 @@ GOTO_INSTRUMENT ?= goto-instrument
 VIEWER ?= cbmc-viewer
 MAKE_SOURCE ?= make-source
 VIEWER2 ?= cbmc-viewer
+CMAKE ?= cmake
+ARPA ?= $(CBMC_ROOT)/aws-proof-build-assistant/arpa
```

Comment:
I think this should be in Makefile-project-defines. You will still be able to use `CBMC_ROOT` because that variable is defined before the local Makefiles are included.

I suggest that you delete this line, and instead write a comment inside Makefile-project-defines suggesting that people define `ARPA`. Something like
```
# Path to Arpa executable
# ARPA = 

```

Reply:
Should I replace line 226 by something like this?
```
ARPA ?= arpa
```
For the case where people forget to define `ARPA` in their local Makefile. This would probably give a more understandable error message too.

Reply:
That's a good idea. I would go even further:
```
ARPA ?= @echo "You must set `ARPA` in Makefile-project-defines to run arpa in this project"; false
```
😃